### ISSUE TITLE
Browser installation does not work - link unactive

### ISSUE THREAD

**Issue by DenadaPatron** [gasolin/aframe-href-component#6]
https://rawgit.com/gasolin/aframe-href-component/master/dist/aframe-href-component.min.js seems to be 404

**Comment by Cryptogenysis** [comment]
Yeah, Ive been struggling with this bug all month.. i guess the gasolin CDN expired :[ 
On the bright side I found a replacement  thnx to Liz Kobrazova's VR business card on CodePen :]
https://res.cloudinary.com/dybluctxg/raw/upload/v1495309810/index_bp9czc.js

If this expires just copy the original JS and host it yourself.
